scout/server/blueprints/cases/templates/cases/utils.html

```diff
@@ -1,5 +1,46 @@
 {% from "variants/utils.html" import tier_cell %}
 
+{% macro variant_transcripts(gene) %}
```

Comment:
Wrote a test for this macro, you can find it under tests/server/blueprints/cases/test_cases_templates.py